src/ledger/LedgerCloseMetaFrame.h

```diff
@@ -38,6 +38,15 @@ class LedgerCloseMetaFrame
 
     LedgerCloseMeta const& getXDR() const;
 
```

Comment:
I think this PR also needs a change to populate refunds in `postTxApplyFeeProcessing` , as the respective change has already been done for the fee events.

Reply:
It'll only be populated in protocol 23 right? Are we expecting anyone to run rc1 upgraded to 23? If not, this can be done in the parallel soroban PR.

Reply:
I think we want rc1 to be used in p23 integration tests.